qa/SidechainTestFramework/scutil.py

```diff
@@ -496,16 +496,17 @@ def is_mainchain_block_included_in_sc_block(sc_block, expected_mc_block):
     return is_mac_block_included
 
 """
-Verify the wallet balance is equal to an expected balance.
+Verify the wallet coins balance is equal to an expected coins balance.
+Note: core coins boxes are: RegularBox and ForgerBox
 
 Parameters:
  - sc_node: a sidechain node
  - expected_wallet_balance
 """
-def check_wallet_balance(sc_node, expected_wallet_balance):
-    response = sc_node.wallet_balance()
+def check_wallet_coins_balance(sc_node, expected_wallet_balance):
```

Comment:
Method check_wallet_coins_balance also needs to be updated in mc_sc_forging_delegation.py file.

Reply:
done